OMCompiler/Compiler/runtime/CMakeLists.txt

```diff
@@ -245,7 +245,7 @@ else()
   set(CONFIG_WITH_OPENMP 0)
 endif()
 
-set(WITH_UMFPACK "#define WITH_UMFPACK")
+set(WITH_SUITESPARSE "#define WITH_SUITESPARSE")
```

Comment:
Just FYI, these config related defines are not complete for the new CMake config. I added them to get a quick working compilation. We need to update the autoconf defines with proper names and consistent #define style. 
`set(...)` to a whole `"#define ..." `is probably not the way to go. Both CMake and autoconf have builtin support for handling these "availability" defines. 
I will try to finish it up but maybe you can take over as well. Someone needs to be responsible for it :)